docker-compose.yml

```diff
@@ -1,8 +1,14 @@
 version: "3.2"
 
 services:
-  redis:
-    container_name: test_redis
+  redis1:
+    container_name: test_redis_1
     image: redis:6-alpine
     ports:
       - "6379:6379"
+
+  redis2:
```

Comment:
Note the general idea here is to test commands that require two Redis connections such as migrate, so the docker compose environment now brings up two servers with different exposed ports.